### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

I'm actually interested in figuring out how to make `fchurn` into an action...

### CODE REVIEW

File: packages/io/src/io.ts [hunk 1]
```diff
@@ -227,7 +227,7 @@ export async function which(tool: string, check?: boolean): Promise<string> {
     // build the list of directories
     //
     // Note, technically "where" checks the current directory on Windows. From a task lib perspective,
-    // it feels like we should not do this. Checking the current directory seems like more of a use
+    // it feels like we should not do this. Check ing the current directory seems like more of a use
```
Comment (damccorm) [inline packages/io/src/io.ts]: I don't think we need this change.

Comment (damccorm) [reply packages/io/src/io.ts]: There's also now conflicts from separate changes - you can probably just resolve using the version from master

Comment (jsoref) [reply packages/io/src/io.ts]: Sorry, global search-and-replace was trying to change `Checkin` not `Checking` but was careless.

**Comment by damccorm** [comment]
Looks great - thanks for the contribution!

**Pull request merged**